lsp-methods.el

```diff
@@ -332,12 +337,6 @@ whitelist, or does not match any pattern in the blacklist."
   :type '(repeat regexp)
   :group 'lsp-mode)
 
-;;;###autoload
-(defcustom lsp-enable-eldoc t
```

Comment:
Removing these options will break setups for people who disable it. We should increment the major version of if we do, and document the change as well.

Reply:
Bump to 4.3

Is it sufficient? I don't know how to document the change.. the wiki is not editable

Reply:
It is straightforward for people who disable it to migrate to `(setq lsp-eldoc-hook nil)`.

Reply:
Since it is a backwards incompatible change, the new version should be 5.0.

Reply:
Done